tests/test_reporter.py

```diff
@@ -0,0 +1,109 @@
+import os
+import pytest
+from unittest.mock import patch, Mock
+
+import yaml
+
+from app.reporter import update_cost_reporter, get_current_and_previous_months_dates
+
+
+@pytest.fixture
+def aws_static_costs():
+    return {"current_month": 700, "previous_month": 800}
+
+
+@pytest.fixture
+def aws_accounts_file():
+    aws_accounts_filename = os.path.join(f"{os.getcwd()}/tests", "test_accounts.yaml")
+    os.environ["AWS_COST_REPORTER_CONFIG"] = aws_accounts_filename
+
+    return aws_accounts_filename
+
+
+@pytest.fixture
+def aws_account_names(aws_accounts_file):
+    with open(aws_accounts_file, "r") as fd:
+        aws_accounts_data = yaml.safe_load(stream=fd)
+        _aws_account_names = aws_accounts_data["accounts"].keys()
+
+        assert _aws_account_names, "Failed to get AWS account names to report costs"
```

Comment:
I don;t understand this assert, you create the file and assert on it?

Reply:
removed